src/utils.py

```diff
@@ -153,6 +164,44 @@ def chrome_browser_options():
 
     return options
 
+def firefox_browser_options():
```

Comment:
can you also move chrome and Firefox to separate files. maybe chrome.py or firefox.py

Reply:
Same for the profiles, the options are now placed under different browser classes. Thanks.